cfgov/unprocessed/js/modules/util/Behavior.js

```diff
@@ -0,0 +1,34 @@
+/* ==========================================================================
+   Get Breakpoint State
+   ========================================================================== */
+
+'use strict';
+
+var Behavior = {};
+
+/**
+ * @param {string} selector Used to query dom for elements.
+ * @param {string} event Event type to add to element.
+ * @param {function} eventHandler Callback for event.
+ */
+Behavior.create = function create( selector, event, eventHandler ) {
+  var behaviorElements = document.querySelectorAll( selector );
+  if( behaviorElements === null ) {
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/2118/commits/e9a3f51306989e9afa4b6dafdf10fc31ec4ba73c